airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -350,6 +350,35 @@ class Algorithm(Enum):
     EdDSA = "EdDSA"
 
 
+class InjectInto(Enum):
```

Comment:
I've been having trouble getting the `poetry run poe build` command to run locally for me right now. I made similar changes to what was noted by David https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1757430835462669 

However, I'm still seeing the same error despite upgrading to dagger 18. I did some shenanigans by writing this by hand so I don't block testing or review. But i'll look into this more tomorrow